src/Symfony/Component/Console/Helper/ProgressBar.php

```diff
@@ -339,15 +339,23 @@ public function start($max = null)
      * Advances the progress output X steps.
      *
      * @param int $step Number of steps to advance
-     *
-     * @throws LogicException
      */
     public function advance($step = 1)
     {
         $this->setProgress($this->step + $step);
     }
 
     /**
+     * Regresses the progress output X steps.
+     *
+     * @param int $step Number of steps to regress
+     */
+    public function regress($step = 1)
```

Comment:
We could do that, but that sounds like an oxymoron calling `advance()` to actually step backwards, adding `regress()` at least keeps the API consistent